### DISCUSSION TITLE
Welcome to computation-ai Discussions!

### DISCUSSION THREAD

**Discussion by aiaa-deic** [aiaa-deic/computation-ai#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Offer constructive feedback, broaden perspectives, and help refine messaging.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about your vision and interest in this community.

**Comment by collinsku** [comment]
Hello everyone! Excited to be part of this community. My name is Kuan Collins. I am the Executive Director for Digital Innovation at a company called Axient. In my current role, I am helping the teams in my company to modernize the delivery of our SE&I, digital engineering and modeling & simulation capabilities with cloud, enhanced customer experience, cATO and Zero Trust Security, employment of AI/ML and infrastructure optimization coupled with employment of DevSecOps, MBSE, and Agile methodologies. Also building digital transformation solutions for our customers in DoD, NASA, AF, Navy, Army, MDA, Space Force, etc. Looking forward to engaging with everyone.

Here is my LinkedIn profile: https://www.linkedin.com/in/collinsku/
